Ryujinx.Graphics.Vulkan/FramebufferParams.cs

```diff
@@ -140,6 +140,25 @@ public Auto<DisposableImageView> GetAttachment(int index)
             return _attachments[index];
         }
 
+        public ComponentType GetAttachmentComponentType(int index)
+        {
+            if (_colors != null && (uint)index < _colors.Length)
```

Comment:
```suggestion
            if (_colors != null && (uint)index < (uint)_colors.Length)
```
Avoid a implicit sign extension to long.

Reply:
Nevermind, checked codegen and it doesn't make any difference.